modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -18,4 +18,47 @@ Depending on your {ibm-z-name} environment, you can choose from the following op
 [NOTE]
 ====
 Currently, ISO boot support on {ibm-z-name} (`s390x``) is available only for {op-system-base-full} KVM, which provides the flexibility to choose either PXE or ISO-based installation. For installations with z/VM and Logical Partition (LPAR), only PXE boot is supported.
-====
\ No newline at end of file
+====
+
+[id="networking-reqs-ibmz_{context}"]
+== Networking requirements for {ibm-z-title}
+
+In {ibm-z-title} environments, advanced networking technologies such as Original Storage Architecture (OSA), HiperSockets, and Remote Direct Memory Access (RDMA) over Converged Ethernet (RoCE) require specific configurations that deviate from the standard settings used in the Agent-Based Installer deployments. These overrides are necessary to accommodate their unique requirements and ensure a successful and efficient deployment on {ibm-z-title}.
+
+The following table lists the network devices that are supported for the network configuration override functionality:
+
+[cols="3,2,2,2,2", options="header"]
+|====
+| Network device 
+| z/VM  
+| KVM 
+| LPAR Classic 
+| LPAR Dynamic Partition Manager (DPM)
+
+| Original Storage Architecture (OSA) virtual switch 
+| Not needed
+| Not applicable
+| Not needed
+| Not needed
+
+| Direct attached OSA 
+| Supported
+| Only through a Linux bridge 
+| Supported
+|  Not needed
+
+| RDMA over Converged Ethernet (RoCE) 
+| Not needed
+| Only through a Linux bridge 
+| Not needed
+| Not needed
+
+| HiperSockets 
+| Supported 
+| Only through a Linux bridge 
+| Supported 
+| Not needed 
+|====
+
+// Configuring network overrides in {ibm-z-title}
+include:modules/configuring-network-overrides-ibmz.adoc[leveloffset=+2]
```

Comment:
This needs to be included in the assembly, not at the bottom of this module. You'd want to place it under [this line](https://github.com/openshift/openshift-docs/blob/main/installing/installing_with_agent_based_installer/prepare-pxe-assets-agent.adoc?plain=1#L39).

Additionally, the include needs an extra colon:
```suggestion
// Configuring network overrides in {ibm-z-title}
include::modules/configuring-network-overrides-ibmz.adoc[leveloffset=+2]
```